templates/field--field-funding-contact-name.html.twig

```diff
@@ -0,0 +1,7 @@
+<div class="funding-ct--summary-section--contact-info-name">
+    {% for item in items %}
+        <span class="wdn-icon-user" aria-hidden="true"></span><span class="wdn-text-hidden">contact icon</span>
```

Comment:
Maybe but maybe not.  At least you wouldn't have the WTF moment of wondering why the second value isn't being displayed.